### PR TITLE
Use `files` in package.json

### PR DESCRIPTION
**Pull request by SimenB**
This avoids getting test-files when installing the package

### CODE REVIEW

**Comment by hypesystem** [comment]
Nice! Don't forget to add yourself to the list of collaborators in package.json and the README.

**Comment by SimenB** [comment]
@hypesystem Updated.

I noticed that `license` was missing from package.json, so I ran `npm init` to add missing fields. It also shortened the form in contributors. Want me to change back?

File: package.json [hunk 1]
```diff
@@ -112,8 +44,12 @@
     "gcm"
   ],
   "main": "index",
+  "files": [
+    "index.js",
+    "lib/"
+  ],
   "scripts": {
-    "test": "sh node_modules/.bin/mocha -R spec test/**/*Spec.js"
+    "test": "mocha test/**/*Spec.js"
```
Comment (SimenB) [inline package.json]: The tests refused to run for me using OSX and `sh`. `spec` is also default reporter.

Comment (hypesystem) [inline package.json]: Unfortunately this doesn't run on Windows per default. I am wondering if there is an obvious cross-platform solution for this ...

**Comment by hypesystem** [comment]
@SimenB the collaborators stuff is fine. I would like a cross-platform (linux/osx/windows) solution for `npm test`, though.

In Windows, the one you have changed it to, requires manually changing PATH.

**Comment by SimenB** [comment]
`npm run` is supposed to handle that... 
I have a window box at home, I'll give it a whack tonight.

**Comment by SimenB** [comment]
Of course, I can remove the changes to test and make a separate PR

**Comment by hypesystem** [comment]
I think ti would be good to make that a separat PR -- so I can merge this asap :)

**Comment by SimenB** [comment]
Sure, I'll fix it tonight

**Comment by SimenB** [comment]
@hypesystem Removed test stuff, see #117

**Comment by hypesystem** [comment]
Thank you!

**Pull request merged**